bucket/foxit-pdf-reader.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "11.0.1.49938",
+    "description": "View, annotate, form fill, and sign PDF across desktop, mobile, and web – no matter if you’re at the office, home, or on the go.",
+    "homepage": "https://www.foxit.com/",
```

Comment:
```suggestion
    "description": "Fast and feature rich PDF reader that offers a delightful reading experience in a small footprint.",
    "homepage": "https://www.foxit.com/pdf-reader/",
```

Reply:
The times when Foxit had a small footprint are over...

Reply:
Homepage is changed. I would suggest to keep the new description.